pkg/conversions/virtualnode_types_conversion.go

```diff
@@ -0,0 +1,329 @@
+package conversions
```

Comment:
The function names in this file is not very go like, it's recommended to use [MixedCaps](https://golang.org/doc/effective_go.html#mixed-caps).

Also, do all these functions have to be public?

Reply:
They don't need to be all public, but I want to follow same naming pattern. (and Public allows to convert specific sub structures if it's needed).
I'm use this naming pattern to make conversion src and dest explicitly, and also follow k8s's pattern.
Quoted from https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#edit-version-conversions
> Since auto-generated conversion functions are using manually written ones, those manually written should be named with a defined convention, i.e. a function converting type X in pkg a to type Y in pkg b, should be named: convert_a_X_To_b_Y.